mlflow/utils/file_utils.py

```diff
@@ -499,7 +499,9 @@ def get_or_create_tmp_dir():
         # Note: For python process attached to databricks notebook, atexit does not work.
         # The /tmp/repl_tmp_data/{repl_id} directory will be removed once databricks notebook
         # detaches.
-        tmp_dir = os.path.join("/tmp", "repl_tmp_data", get_repl_id())
+        # The repl_tmp_data directory is designed to be used by all kinds of applications,
```

Comment:
Can you elaborate on `designed to be used by all kinds of applications`?

Reply:
e.g., on databricks, any applications can write data into `/tmp/repl_tmp_data/{repl_id}` directory, to prevent path conflict, we'd better assign a child directory for each application.
and when notebook detached, the root directory `/tmp/repl_tmp_data/{repl_id}` is automatically deleted.